go/libraries/doltcore/table/editor/sessioned_table_editor.go

```diff
@@ -299,7 +299,8 @@ func (ste *sessionedTableEditor) onDeleteHandleRowsReferencingValues(ctx context
 				}
 			}
 		case doltdb.ForeignKeyReferenceOption_DefaultAction, doltdb.ForeignKeyReferenceOption_NoAction, doltdb.ForeignKeyReferenceOption_Restrict:
-			indexKeyStr, _ := types.EncodedValue(ctx, indexKey)
+			value, _ := indexKey.Get(1)
```

Comment:
You should use the formatKey() function for this, here and in the other places you do it

Reply:
Sorry, I am not very familiar with Go - how would I use it?

Reply:
```go
indexKeyStr, _ := formatKey(ctx, indexKey)
return sql.ErrForeignKeyChildViolation.New(foreignKey.Name, foreignKey.TableName, foreignKey.ReferencedTableName, indexKeyStr)
```

I am using it like this, but in the output it shows as `["green"]`, not `"green"` for some reason. Indexing it (`indexKeyStr[0]`) doesn't give the first value as expected - instead `%!s(uint8=91)` is output.

Reply:
You should use formatKey, and we'll evolve how it prints keys over time. Right now it will put keys in square brackets, so make the test expectations match